HISTORY.rst

```diff
@@ -1,13 +0,0 @@
-=======
-History
-=======
-
-0.0.1a1 (2016-08-12)
---------------------
-
-* Planning release on PyPI.
-
-0.0.1dev1 (2016-08-25)
-----------------------
-
-* Development (pre-alpha) release on PyPI.
```

Comment:
The purpose of `HISTORY.rst` was to log important changes. It has not been used so far as it was judged unnecessary meanwhile the driver is under the kind of development it was under. We will need it soon probably, especially if we introduce breaking changes, or significant changes. 

So what is the plan now?